docroot/themes/humsci/su_humsci_admin/su_humsci_admin.theme

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * su_humsci_admin.theme
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function su_humsci_admin_form_user_login_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
OH, can this be submitted to the 8.x version of stanford_ssp. This is the exact kind of thing we want in that project.

Reply:
i looked at stanford_ssp... were you trying to influence me to build the 8.x version? 🤔 haha